lib/media/track/remotevideotrack.js

```diff
@@ -165,6 +171,50 @@ class RemoteVideoTrack extends RemoteMediaVideoTrack {
     return this;
   }
 
+  unObserverPIP(el) {
```

Comment:
nit: how about `unObservePip`? That way it's consistent with how you name other Pip related things in this PR. Same for it's counterpart below, `observePip`.

Reply:
Thanks @charliesantos, I have now renamed the functions + added some basic unit test and added a changelog entry.